### PR TITLE
Update Bicicoruña feed

### PR DESCRIPTION
**Pull request by Altonss**

### CODE REVIEW

File: pybikes/data/gbfs.json [hunk 1]
```diff
@@ -1074,6 +1074,17 @@
             },
             "feed_url": "https://belfort-gbfs.klervi.net/gbfs/gbfs.json"
         }
+        {
```
Comment (eskerda) [inline pybikes/data/gbfs.json:1077]: invalid JSON, add comma here

File: pybikes/data/gbfs.json [hunk 2]
```diff
@@ -1074,6 +1074,17 @@
...
+            "tag": "bicicorunha",
+            "meta": {
+                "name": "Bicicoruña",
+                "city": "A Coruña",
+                "latitude": 43.3623,
+                "longitude": -8.4115,
+                "country": "ES"
+            },
+            "feed_url": "https://acoruna.publicbikesystem.net/customer/gbfs/v2/gbfs.json"
+        },
```
Comment (eskerda) [inline pybikes/data/gbfs.json:1087]: invalid JSON, remove this comma

**Review by eskerda — CHANGES_REQUESTED** [review]

**Comment by Altonss** [comment]
Sorry for the typos :/

**Comment by eskerda** [comment]
Cool! Thanks @Altonss

**Pull request merged**